### PR TITLE
Feature/reduced sparql algebra 2

### PR DESCRIPTION
**Pull request by BCommeine**

### CODE REVIEW

File: packages/actor-abstract-bindings-hash/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export * from './lib/AbstractBindingsHash';
+export * from './lib/AbstractBindingsHash';
```
Comment (rubensworks) [inline packages/actor-abstract-bindings-hash/index.ts]: This looks unneeded :-)

File: packages/actor-init-sparql/config/sets/sparql-queryoperators.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
   "@context": [
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-init-sparql/^1.0.0/components/context.jsonld",
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/runner/^1.0.0/components/context.jsonld",
-
```
Comment (rubensworks) [inline packages/actor-init-sparql/config/sets/sparql-queryoperators.json]: Please undo this line removal. This is here to logically separate similar packages.

File: packages/actor-init-sparql/config/sets/sparql-queryoperators.json [hunk 3]
```diff
@@ -92,11 +92,13 @@
...
-      "caqodh:hashAlgorithm": "sha1",
-      "caqodh:digestAlgorithm": "base64"
+      "cbqo:mediatorQueryOperation": { "@id": "config-sets:sparql-queryoperators.json#mediatorQueryOperation" }
+    },
+    {
+      "@id": "config-sets:sparql-queryoperators.json#myReducedQueryOperator",
+      "@type": "ActorQueryOperationReducedHash",
+      "cbqo:mediatorQueryOperation": { "@id": "config-sets:sparql-queryoperators.json#mediatorQueryOperation" }
     },
-
```
Comment (rubensworks) [inline packages/actor-init-sparql/config/sets/sparql-queryoperators.json]: Also restore the empty line here.

File: packages/actor-query-operation-reduced-hash/components/Actor/QueryOperation/ReducedHash.jsonld [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  "@id": "npmd:@comunica/actor-query-operation-reduced-hash",
+  "components": [
+    {
+      "@id": "caqorh:Actor/QueryOperation/ReducedHash",
+      "@type": "Class",
+      "extends": "caabh:Actor/AbstractBindingsHash",
+      "requireElement": "ActorQueryOperationReducedHash",
+      "comment": "A comunica Reduced Hash Query Operation Actor.",
+      "parameters": [
+          {
```
Comment (rubensworks) [inline packages/actor-query-operation-reduced-hash/components/Actor/QueryOperation/ReducedHash.jsonld]: We use indentation of 2 characters, while this indents 4 characters.

File: packages/actor-abstract-bindings-hash/index.ts (hunk 1 above)
Comment (BertCommeine) [reply packages/actor-abstract-bindings-hash/index.ts]: Can't be sure enough

**Comment by rubensworks** [comment]
You just removed even more empty lines, please don't :-)

Also, your JSON seems to be invalid now.

**Pull request merged**